resources.whatwg.org/browser-logos/chrome.svg

```diff
@@ -1 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="1 1 190 190"><defs><path id="e" d="M8 184h83.8l39-39v-29H61.2L8 24.5z"/><path id="g" d="M96 56l34.6 60L92 184h92V56z"/><path id="c" d="M22 8v108h39.4L96 56h88V8z"/></defs><linearGradient id="d" x1="29.3" x2="81.8" y1="75" y2="44.4" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#a52714" stop-opacity=".6"/><stop offset=".7" stop-color="#a52714" stop-opacity="0"/></linearGradient><linearGradient id="f" x1="110.9" x2="52.5" y1="164.5" y2="130.3" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#055524" stop-opacity=".4"/><stop offset=".3" stop-color="#055524" stop-opacity="0"/></linearGradient><linearGradient id="h" x1="121.9" x2="136.6" y1="49.8" y2="114.1" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#ea6100" stop-opacity=".3"/><stop offset=".7" stop-color="#ea6100" stop-opacity="0"/></linearGradient><radialGradient id="a" cx="668.2" cy="56" r="84.1" gradientTransform="translate(-576)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#3e2723" stop-opacity=".2"/><stop offset="1" stop-color="#3e2723" stop-opacity="0"/></radialGradient><radialGradient id="i" cx="597.9" cy="48.5" r="78" xlink:href="#a"/><radialGradient id="j" cx="95.8" cy="96.1" r="87.9" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#263238" stop-opacity=".2"/><stop offset="1" stop-color="#263238" stop-opacity="0"/></radialGradient><radialGradient id="k" cx="34.3" cy="32" r="176.8" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#fff" stop-opacity=".1"/><stop offset="1" stop-color="#fff" stop-opacity="0"/></radialGradient><clipPath id="b"><circle cx="96" cy="96" r="88"/></clipPath><g clip-path="url(#b)"><use fill="#db4437" xlink:href="#c"/><use fill="url(#d)" xlink:href="#c"/><use fill="#0f9d58" xlink:href="#e"/><use fill="url(#f)" xlink:href="#e"/><use fill="#ffcd40" xlink:href="#g"/><use fill="url(#h)" xlink:href="#g"/><g fill-opacity=".1"><path fill="#3e2723" d="M62.3 115.7L22.5 47.4l-.6 1 39 67.8z"/><path fill="#263238" d="M129.8 117.3l-.8-.4-38.3 67h1l38.3-67z"/></g><path fill="url(#a)" d="M96 56v21l78.4-21z"/><path fill="url(#i)" d="M22 48.5l57.2 57.2L61.4 116z"/><path fill="url(#j)" d="M91.8 184l21-78.3 17.8 10.3z"/><circle cx="96" cy="96" r="40" fill="#f1f1f1"/><circle cx="96" cy="96" r="32" fill="#4285f4"/><g fill="#fff" fill-opacity=".1"><path d="M130.6 116a40 40 0 0 1-69.2 0L8 24.5v1L61.4 117a40 40 0 0 0 69.2 0z"/><path fill-opacity=".2" d="M131 117.3a39.3 39.3 0 0 0 3.4-32 38 38 0 0 1-3.8 30.7L92 184h1l38.2-66.5zM96 9a88 88 0 0 1 88 87.5V96A88 88 0 0 0 8 96v.5A88 88 0 0 1 96 9z"/><g fill="#3e2723"><path fill-opacity=".2" d="M96 55c-22 0-40 18-40 40v1c0-22 18-40 40-40h88v-1z"/><path d="M97 56h-.5a40 40 0 1 1 0 80h.5c22 0 40-18 40-40s-18-40-40-40z"/><path d="M96 183a88 88 0 0 0 88-87.5v.5A88 88 0 0 1 8 96v-.5A88 88 0 0 0 96 183z"/></g></g></g><circle cx="96" cy="96" r="88" fill="url(#k)"/></svg>
```

Comment:
We should probably not check in these files at all, since they are generated during deploy.

Reply:
Done.